### ISSUE TITLE
Resizable Column only applies to the row without the Header Column

### ISSUE THREAD

**Issue by BoomCheeseBurge** [wenzhixin/bootstrap-table#6907]
### Description

Hello! I would really appreciate the help on this one. I am currently trying to enable the `resizable` feature onto my Bootstrap Table table. I am using the latest version of Bootstrap-Table which is version `1.22.1`. I have also imported the necessary CSS and JS files as per the Bootstrap-Table documentation (albeit quite poorly documented for beginners like me). I am not sure if the issue occurs due to how I defined the table columns in JS as the following.

JS Code

```
var $table = $('#table')
var selections = []

function detailFormatter(value, row, index) {
    return [
		'<button type="button" class="btn btn-warning" data-bs-toggle="modal" data-bs-target="#editModal">',
		'Edit',
		'</button>'
    ].join('')
  }


function initTable() {
	var icons = {
		columns: 'bi-layout-sidebar-inset-reverse',
		fullscreen: 'bi-arrows-fullscreen'
	}
	$table.bootstrapTable('destroy').bootstrapTable({
		icons: icons,
		height: 550,
		locale: 'en-US',
		columns: [
		{
			title: 'ID',
			field: 'id',
			align: 'center',
			valign: 'middle'
		},{
			title: 'Name',
			field: 'name',
			align: 'center',
			sortable: true,
			align: 'center'
		  }, {
			title: 'E-mail',
			field: 'email',
			align: 'center',
			valign: 'middle',
		  }, {
			title: 'Phone',
			field: 'phone',
			align: 'center',
			valign: 'middle'
		  }, {
			  title: 'Location',
			  field: 'location',
			  align: 'center',
			  valign: 'middle',
			  sortable: true
		}, {
			title: 'View',
			field: 'view',
			align: 'center',
			switchable: 'false',
		    formatter: detailFormatter
	  }]
	})
}

$(function() {
	initTable()
})
```

And the following is the table defined in my HTML file.

HTML Code

```
	<table
	id="table"
	data-toolbar="#toolbar"
	data-search="true"
	data-advanced-search="true"
	data-show-refresh="true"
	data-show-fullscreen="true"
	data-show-columns="true"
	data-show-columns-toggle-all="true"
	data-show-export="true"
	data-minimum-count-columns="2"
	data-pagination="true"
	data-id-field="id"
	data-page-list="[10, 25, 50, 100, all]"
	data-url='somejsonfile'
          data-resizable="true">
	</table>
```

Most importantly, the following is my imported dependencies required for the resizable feature to work.

	<link rel="stylesheet" href="/path/to/bootstrap-5.3.0/css/bootstrap.css">

	<link rel="stylesheet" href="/path/to/jquery-resizable-columns-gh-pages/dist/jquery.resizableColumns.css">

	<link rel="stylesheet" href="/path/to/bootstrap-table/dist/bootstrap-table.min.css">

	<script src="/path/to/bootstrap-5.3.0/js/bootstrap.bundle.js"></script>

	<script src=/path/to/js/jquery-3.7.0.min.js"></script>

	<script src="/path/to/store.js-master/storages/localStorage.js"></script>

	<script src="/path/to/jquery-resizable-columns-gh-pages/dist/jquery.resizableColumns.min.js"></script>

	<script src="/path/to/js/tableExport.min.js"></script>
	<script src="/path/to/bootstrap-table/dist/bootstrap-table.min.js"></script>
	<script src="/path/to/bootstrap-table/dist/extensions/export/bootstrap-table-export.min.js"></script>
	<script src="/path/to/bootstrap-table/dist/extensions/resizable/bootstrap-table-resizable.min.js"></script>

For some reason, the resizable feature works on the table. Unfortunately, I can only resize the row of the table excluding the header column. Is there any idea why this is happening? Thanks.

**Comment by wenzhixin** [comment]
Please provide an [Online Example](https://live.bootstrap-table.com) to show your problem. Thanks!

**Comment by BoomCheeseBurge** [comment]
I apologize for interrupting your time. Apparently, it is my mistake. I saw the warning on the documentation before I started importing the necessary dependencies and code for the resizable extension.

![image](https://github.com/wenzhixin/bootstrap-table/assets/80831628/417c13f5-3840-42c6-9793-12e3dfde67ef)

However, I disregarded the height of the table being set in my JS file `height: 550,` since I initially copied the bootstrap-table code from the example in the documentation and didn't pay attention more on the height value until I wrote the code again on the Online Example of yours. I was too caught up thinking the issue may have come from one (or more) of the imported dependencies being wrong. Now, it works! Thanks.

**Comment by wenzhixin** [comment]
> Now, it works! 

Great! I will close this Issue. Thanks for your feedback!

**Comment by BoomCheeseBurge** [comment]
Thank you for the quick response! I apologize for interrupting your time. Apparently, it is my mistake. I saw the warning on the documentation before I started importing the necessary dependencies and code for the resizable extension.

![image](https://github.com/wenzhixin/bootstrap-table/assets/80831628/417c13f5-3840-42c6-9793-12e3dfde67ef)

However, I disregarded the height of the table being set in my JS file `height: 550,` since I initially copied the bootstrap-table code from the example in the documentation and didn't pay attention more on the height value until I wrote the code again on the Online Example of yours. I was too caught up thinking the issue may have come from one (or more) of the imported dependencies being wrong. Now, it works! Thanks.

**Comment by BoomCheeseBurge** [comment]
Thank you for the quick response! I apologize for interrupting your time. Apparently, it is my mistake. I saw the warning on the documentation before I started importing the necessary dependencies and code for the resizable extension.

![image](https://github.com/wenzhixin/bootstrap-table/assets/80831628/417c13f5-3840-42c6-9793-12e3dfde67ef)

However, I disregarded the height of the table being set in my JS file `height: 550,` since I initially copied the bootstrap-table code from the example in the documentation and didn't pay attention more on the height value until I wrote the code again on the Online Example of yours. I was too caught up thinking the issue may have come from one (or more) of the imported dependencies being wrong. Now, it works! Thanks.
